MicroAOD/python/MicroAODCustomize.py

```diff
@@ -178,15 +178,15 @@ def customize(self,process):
         if self.runDec2016Regression:
             self.customizeDec2016Regression(process)
         if self.runEGMEleID:
-            #if 'eleIdVersion' in self.metaConditions.keys():
-            getattr(self, 'customize'+self.metaConditions['eleIdVersion'])(process)            
-            # else:
-            #     raise Exception, "runEGMEleID set to True but no eleIdVersion specified in conditions json file"
+            if 'eleIdVersion' in self.metaConditions.keys():
```

Comment:
Is this for back-compatibility in case one wants to run the old V1 and 2016 IDs?

Reply:
yes, if one adds a line in the metaconditions can customize this. i thought that someone might want to reproduce the old result.